fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -365,9 +365,7 @@ private void refreshCardinality() {
             Set<ApiFilter> filters,
             PaginationParameters paginationParameters
     ) {
-        int perPage = paginationParameters.getPerPage();
-        validatePerPage(perPage);
```

Comment:
The methods called do use the `paginationParameters` object, so we still need to do the validation. If the `getResultsPage` method _also_ does the validation, we can drop it from here, but if not, we should keep this.